portal/src/main/webapp/WEB-INF/jsp/index.jsp

```diff
@@ -169,7 +169,13 @@ if (sessionError != null) {  %>
                 }
             %>
             <%@ include file="query_form.jsp" %>
-
+            <script type="text/javascript">
+                $(document).ready(function() {
+                    if (window.cancer_study_id_selected.length > 0 && window.cancer_study_id_selected !== "null") {
```

Comment:
In either cases, there's no way the exception could ever happen - but yea, it doesn't make sense. (The code here is really ancient..)